src/components/upload.js

```diff
@@ -26,42 +27,97 @@ const Clickable = styled.div.attrs({
   max-height: 370px;
 `
 
+const ImagesSidebyside = styled.div.attrs({
+  className: "flex flex-column-ns flex-row bg-light-pink",
+})``
+
+const Image = styled.img.attrs({ className: "w-40-ns w-90" })`
+  max-width: 483px;
```

Comment:
I think we'll need to make this a div with set width and height relative to the viewport/container and adjust for different devices, then use the image as background-image with display: cover, so that the images sizes don't change around.